packages/atomic/src/components/common/smart-snippets/atomic-smart-snippet-feedback-modal/feedback-options.spec.ts

```diff
@@ -0,0 +1,69 @@
+import {describe, expect, it} from 'vitest';
+import {feedbackOptions} from './feedback-options';
+
+describe('feedbackOptions', () => {
```

Comment:
feedback-options

but again this is a new pattern, so not quite sure how we should name the top-level describe 😂 Seems like naming it after the module (like we would for a utils) is appropriate.